Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM alpine:3.6
-ENV ETCD_VER v3.0.17
+FROM alpine:3.8
+
+ENV ETCD_VER v3.3.9
```

Comment:
do we run that version everywhere now? I presume it's backward compatible with all v3.x?

Reply:
Yeah it's backwards compatible, was just for some of the env var changes.